src/components/ProductCard.component/ProductCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const ProductCard = () => {
+  return (
+    <div className="card card--shadow position-r  text-sm txt-algn">
+      <div className="badge card-badge heart-badge-active position-a br-md color-white">
+        <i className="fas fa-heart"></i>
+      </div>
+      <div className="card__img">
+        <img
+          className="img-100"
```

Comment:
use same naming conventions for className.

Reply:
Okay